src/global.c

```diff
@@ -0,0 +1,6 @@
+
+#define REDISMODULE_MAIN
```

Comment:
Why do we need this file?

Reply:
`global.c` is excluded in Makefile from the `-include $(SRCDIR)/common.h`, therefore allowing pre-defining `REDISMODULE_MAIN`.